sahi/utils/cv.py

```diff
@@ -380,9 +380,9 @@ def visualize_object_predictions(
         Path(output_dir).mkdir(parents=True, exist_ok=True)
         # save inference result
         save_path = os.path.join(output_dir, file_name + "." + export_format)
-        cv2.imwrite(save_path, cv2.cvtColor(image, cv2.COLOR_RGB2BGR))
+        
     elapsed_time = time.time() - elapsed_time
-    return {"image": image, "elapsed_time": elapsed_time}
+    return {"image": image, "elapsed_time": elapsed_time}, image
```

Comment:
Why changing return type from dict to tuple, and we already have `image` in the return dict, why we need an extra ?

Reply:
You are right. I didn't pay attention to return